tests/unittests/RecommendationSystemTest.cpp

```diff
@@ -745,40 +761,46 @@ class RecommendationSystemTest : public BackendTest {
   /// Create partitions to run and compare results.
   void runPartitionedGraph(size_t numDevices, size_t memSize,
```

Comment:
Do we still need this? Can we just add it to the EE and let it do its thing?

For context this was added originally because we didn't have a nice wrapper of the runtime to test partitioning with, and now we do.

Reply:
We still do, this function calls partitioner directly for finer grained testing. That's not something the EE exposes. An alternative would be to go straight to HostManager and bypass EE but that would require its own supporting code.